src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/config.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+  "properties": {
```

Comment:
Since some of these aren't actually properties, might want to pick a word like "options" or something, so it's not weird to find targets inside this collection